docs/releasing.md

```diff
@@ -146,6 +146,23 @@ BMO e2e is running as GitHub Actions. We need to add released branch and remove
 the non-maintained branches there, along with the suitable configurations with
 recently released Ironic-image releases as well in the fixtures.
 
+### Dependabot configuration
+
+In `main` branch, we need to update Dependabot configuration to allow updates
```

Comment:
I would use either "you" as or "one, the release team etc..." instead of the "we". This "we" was introduced in the "Release artifacts" section and doesn't really fit into a technical document or to a instruction guide.

Reply:
Done.